.husky/pre-push

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
+set -e
+node ./scripts/tasks/check-and-rewrite-package-json.js --test
+node ./scripts/tasks/generate-license-files.js --test
+node ./scripts/tasks/verify-treeshakable.js ./packages/@lwc/shared/dist/index.js
+node ./scripts/tasks/check-imports-are-declared-dependencies.js
```

Comment:
These 4 commands take 1.4s on my machine... that's pretty acceptable for a prepush. And I don't believe any of these rely on `yarn build`.